docs/organizations/using-coding-standards.md

```diff
@@ -21,49 +22,74 @@ To create a coding standard for your organization:
 
 1.  Click the button **Create new standard** at the top right-hand corner of the page. This opens the coding standard creation form.
 
-1.  Enter a unique name and click **Create standard**.
+1.  Enter a unique name for the new coding standard.
 
-    Optionally, select a repository to use as a baseline for the new coding standard. This is useful if you already have a well-configured repository that you wish to use as a template.
+    Optionally, you can:
+
+    -   Select a repository to use as a baseline for the new coding standard. This is useful if you already have a well-configured repository that you wish to use as a template.
+    -   Set the new coding standard as your organization's [default coding standard](#set-default), so it applies automatically to new repositories.
 
     ![Creating a new coding standard](images/coding-standard-create.png)
 
-1.  Select the programming languages that the new coding standard should cover. 
+1.  Click **Create coding standard**.
+
+1.  Adjust the sensitivity of the coding standard from **Essential** to **Comprehensive** for each issue category group:
+
+    -   **Essential**: Focuses on the most important issues by including the most critical code patterns. Ideal for initial integration, to prevent overwhelming developers, and to speed up analysis times.
+    -   **Comprehensive**: Detects all issues by including all code patterns, capturing the complete severity spectrum. Ideal for detailed code quality reviews, while reducing emphasis on critical issues. It may increase analysis times.
```

Comment:
@codacy/alabama  Should we mention more reference information here, such as [what the category groups map to](https://codacy.slite.com/app/docs/k8CfoW737FV-27)?

I'm unsure about including this info. Thoughts?

Reply:
Let's not include it for now. But we may revisit it and include it. We may still need to change and tweak what we have as well